<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>140</y>
     <width>291</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 11pt;
background: #01BFFF;
color:white;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 11pt;
background: #01BFFF;
color:white;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Add New</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24pt;
background: #01BFFF;
color:white;
border-radius:8px;</string>
   </property>
   <property name="text">
    <string>Daily Task Planner</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>460</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 11pt;
background: #01BFFF;
color:white;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Delete Item</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>481</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Calendar</string>
    </attribute>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px;
</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Calculator</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>341</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-radius:10px;
background-color : 'lightblue';
                             

</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: 'light blue';
border-radius:10px;
</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: 'light blue';
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: 'light blue';
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: 'light blue';
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: 'light blue';
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: 'light blue';
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: 'light blue';
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: 'light blue';
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_0">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: 'light blue';
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_clear">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: 'light blue';
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_divide">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 190, 111);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_multiply">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 190, 111);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addition">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>210</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 190, 111);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_subtract">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>260</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 190, 111);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_equal">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>260</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 190, 111);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_calculator">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>381</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 221, 218);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Journal</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit_journal">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>471</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_journalChanges">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 11pt;
background: #01BFFF;
color:white;
border-radius:10px;</string>
     </property>
     <property name="text">
      <string>Save Journal Changes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_newEntry">
     <property name="geometry">
      <rect>
       <x>328</x>
       <y>0</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 11pt;
background: #01BFFF;
color:white;
border-radius:10px;</string>
     </property>
     <property name="text">
      <string>Add New Entry</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
